draft-iyengar-quic-delayed-ack.md

```diff
@@ -349,6 +327,43 @@ flight at any given time, this extension does not prohibit having more than one
 in flight. Generally, when using `max_ack_delay` for PTO computations, endpoints
 MUST use the maximum of the current value and all those in flight.
 
+# Implementation Considerations {#implementation}
+
+There are tradeoffs inherent in a sender sending an ACK_FREQUENCY frame to the
+receiver. As such it is recommended that implementers experiment with different
+strategies and find those which best suit their applications and networks.
+There are, however, noteworthy considerations when devising strategies for
+sending ACK_FREQUENCY frames.
+
+## Loss Detection {#loss}
+A sender relies on receipt of acknowledgements to determine the amount of data
+in flight and to detect losses, specifically when packets experience reordering,
+see {{QUIC-RECOVERY}}.  Consequently, how often a receiver sends acknowledgments
+determines how long it takes for losses to be detected at the sender.
+
+## New Connections {#new-connections}
+Many congestion control algorithms have a special state for the begginning
+phases of a connection.  It is typical that in this phase the congestion
+controller will gradually increase the amount of data in the network until it is
+signalled to stop.  While the mechanism used to achieve this increase varies,
+acknowledgments by the peer are generally critical during this phase to drive
+the congestion controller's machinery.  Hence, it is advisable that caution is
+exercised in sending an ACK_FREQUENCY frame to the receiver while the sender's
+congestion controller is in this state.  Delaying acknowledgments excessively
+can lead to increased delay in the sender's congestion controller reaching an
+optimal value, leading to poor performance.  A mechanism to avoid this would be
+for the sender to simply avoid sending an ACK_FREQUENCY until its congestion
+controller exits the state where it is reliant on a high frequency of
+acknowledgments.
```

Comment:
I agree with the premise, but not with the recommendation. A sender might want to ask the receiver to ack every packet during this phase, and that needs an ACK_FREQUENCY frame. I think what you want to say here is not to advise caution, but to note that a sender could use the frame to make the receiver ack more frequently during this phase.